Sources/MongoSwift/WriteConcern.swift

```diff
@@ -128,7 +128,7 @@ public class WriteConcern: Codable {
             let journalStr = String(describing: journal)
             let wStr = String(describing: w)
             let timeoutStr = String(describing: wtimeoutMS)
-            throw MongoError.writeConcernError(message:
+            throw MongoError.writeConcernError(code: 0, message:
```

Comment:
I assume this is something we can clear up with [SWIFT-145](https://jira.mongodb.org/browse/SWIFT-145)? Ideally, we should use something like an invalidArgumentError here and save writeConcernError for an actual runtime exception.